app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = MerchantFacade.merchants
+  end
+
+  def show
+    @merchant = MerchantFacade.merchant(params[:id])
+    @items = MerchantFacade.items(params[:id])
```

Comment:
(low priority)
It took me a second to understand why we're passing in the same ID for methods that return different types of poros. Could this method be called `merchant_items` for specificity? Or could the method accept a keyword argument, making this line something like `@items = MerchantFacade.items(merchant_id: params[:id])`?